jhove-modules/html-hul/src/main/java/edu/harvard/hul/ois/jhove/module/HtmlModule.java

```diff
@@ -446,11 +450,7 @@ public int parse(InputStream stream, RepInfo info, int parseIndex) {
 			HtmlDocDesc docDesc = null;
 			switch (type) {
 			case HTML_3_2:
-			default:
```

Comment:
@nvanderperren I'm missing something with this deletion? A quick explanation for the hard of understanding please.

Reply:
the null values are not necessary because it's not used nor checked. Only the values that are assigned in the if or switch statements are used. But I was maybe what to eager.